curriculum/challenges/english/blocks/workshop-discount-calculator/68e5293bd00d2fe134f58985.md

```diff
@@ -0,0 +1,108 @@
+---
+id: 68e5293bd00d2fe134f58985
+title: Step 11
+challengeType: 20
+dashedName: step-11
+---
+
+# --description--
+
+Since `PercentageDiscount` inherits from an abstract class, you must implement all abstract methods. Start by implementing the `is_applicable` method.
```

Comment:
please mention that it should have the same parameters and type hints as in the abstract class, as this is the first step in which it is asked